zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -5712,6 +5716,24 @@ components:
         name:
           type: string
           description: The updated display name of the group.
+        description:
+          type: string
+          description: The updated description of the group.
```

Comment:
🔧 Can we just get rid of this changeset? This was introduces for partial updates, but since we won't support those it's only extra nesting.

Reply:
I plan to do this refactor in a separate issue, as it might have impact also on the client and probably in the processor 👍